_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,64 @@
+The banner component supports both responsive images and adaptive images. Responsive images are simply driven by CSS rules, which allow you to reuse the same images across different devices and screen sizes. Adaptive images can be used simultaneously, and are driven by the images provided by the back end. The back end provides different media items according to the media format. The `cx-media` component adds the different formats in a `srcset` so that the browser can decide when to use which image. Media items are rendered in the `cx-media` component. For more information, see [Media Component]({{ site.baseurl }}{% link _pages/dev/components/shared-components/media-component.md %}).
 
 ## Component Data
 
 The banner component renders the following properties:
 
-| config     | description                                                                                                         |
-| ---------- | ------------------------------------------------------------------------------------------------------------------- |
-| `urlLink`  |                                                                                                                     |
-| `external` | If set to true, the URL will not use the Angular routerLink, but an ordinary href instead.                          |
-| `media`    | The media contains different media per format, so that an optimzed media can be used for the given space and usage. |
-| `headline` | The headline is rendered as paragraph tag before the media.                                                         |
-| `content`  | The content is rendered in a paragraph tag below the media.                                                         |
+| Config     | Description                                                                                                          |
+| --- | --- |
+| `urlLink` | An optional link to an internal or external resource. |
+| `external` | If set to true, the URL will not use the Angular routerLink, but an ordinary href instead. |
+| `media` | The `media` contains different media items for each format, so that an optimized media item can be used for the given space and usage. |
+| `headline` | The `headline` is rendered as a paragraph tag before the media. |
+| `content` | The `content` is rendered in a paragraph tag below the media. |
+
+**Note:** Some of these properties might not be available, depending on the banner component type you receive from the back end.
 
 ## Component Logic
 
-There's no component logic in the banner component, the component just renders component data. You can use the component specific `CmsBannerComponent` to access the component data.
+There is no component logic in the banner component. The component simply renders component data. You can use the component-specific `CmsBannerComponent` to access the component data.
```

Comment:
You can use the injected component data (of type `CmsBannerComponent`) to access the component data.

Reply:
👍